src/odemis/acq/feature.py

```diff
@@ -119,3 +128,60 @@ def load_project_data(path: str) -> dict:
             f.streams.value.extend(data_to_static_streams(open_acquisition(fname)))
 
     return {"overviews": overview_data, "features": features}
+
+
+def import_features_from_autolamella(path: str) -> List[CryoFeature]:
+    """Import feature positions from an autolamella experiment
+    :param path (str): path to the autolamella experiment directory
+    :return (List[CryoFeature]): list of CryoFeature
+    """
+
+    with open(os.path.join(path, "experiment.yaml"), "r") as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+
+    # get the relevant components
+    linked_yz_stage = model.getComponent(name="Linked YZ")
+    pm = MicroscopePostureManager(model.getMicroscope())
+
+    cryo_features = []
+    for lamella in data["positions"]:
```

Comment:
I'm not sure if performance is a problem, but I think a lot can be improved with vectorization. I'm also mainly talking about existing code. Functions like `getCurrentPostureLabel` and especially `_transformFromSEMToMeteor` have a lot of redundancy when called in a loop. If we'd allow these functions to accept multiple positions and implement vectorization internally, we can shave of a lot of time.

Reply:
By the way, way out of scope for now :smile: !

Reply:
not a bottleneck currently